### DISCUSSION TITLE
Jailbreak iphone

### DISCUSSION THREAD

**Discussion by hookne** [community/community#65711] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Are the questions posted here?
 I tried dualra1n to downgrade the ios version But when the phone restarts, it works on the modern version. Is there a way when the phone restarts Disable startup Double boot

**Comment by Akash1134** [comment]
Hi @hookne ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
